image-loader/app/lib/FailureResponse.scala

```diff
@@ -29,4 +29,13 @@ object FailureResponse extends ArgoHelpers {
       s"Unsupported file type: not an image type. Supported: ${supportedMimeTypes.mkString(", ")}"
     )
   }
+  def badImage(supportedMimeTypes: List[String]): Result = {
```

Comment:
Could this be `unsupported` rather than `bad`? `bad` could mean a corrupted file, for example.